feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -2,7 +2,7 @@ export default class DataAccount {
     constructor() {
         this.importingResults = [];
         this.importedFileNames = [];
-        this.name = "";
+        this.personalData = { name: { givenName: "", familyName: "" } };
```

Comment:
No middle name? Or would that be included into given name? Also, people can have several family names (like us) so maybe plural?

Reply:
This is tied to schema.org's rdf vocab, where they went for givenName and familyName. I think for spaniards and other multi family name people the combination of the names is the name? Does that make sense? (Also my point here was to use the same attr names as schema)

Reply:
I might argue that then they should be plural, but if they are not, I'm OK with that either, even more of if it's the same thing that it's used in schema.org